src/app/search-users/search-users.component.ts

```diff
@@ -94,6 +99,22 @@ export class SearchUsersComponent implements OnInit {
     });
   }
 
+  downloadAnonymousUserProfile(userReference: string): void {
+    this.downloadService.downloadAnonymousUserProfile(userReference).subscribe((data: Blob) => {
+
```

Comment:
Is it possible to use here [file-saver](https://www.npmjs.com/package/file-saver)?

Reply:
Thank you for the idea! Yes, it is possible. Done.